packages/backend/tests/api/api.test.ts

```diff
@@ -0,0 +1,38 @@
+import app from 'app';
+import supertest from 'supertest';
+
+const request = supertest;
+
+describe('Api', () => {
+    it('should return a mug in a list of goodies', async () => {
+        const result = await request(app).get('/api/goodies').expect(200);
+        expect(result.body).toContainEqual(expect.objectContaining({ name: 'Mug' }));
+    });

```

Comment:
this test is not necessary, it can be removed

Reply:
test has been removed